src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -24,6 +28,11 @@ internal void InitializeLicense(string licenseText)
 
             if (string.IsNullOrWhiteSpace(licenseText))
             {
+                if (FirstTimeUser() && (Debugger.IsAttached && SystemInformation.UserInteractive))
+                {
+                    TrackFirstTimeUsageEvent();
+                    SetupFirstTimeRegistryKeys();
```

Comment:
since `TrackFristTimeUsageEvent` is `async void`, we will also setup the registry key even if the "tracking" failed. Is that what we want?